code/lib/ui/package.json

```diff
@@ -99,5 +98,9 @@
       "./src/runtime.ts"
     ]
   },
-  "gitHead": "fc90fc875462421c1faa35862ac4bc436de8e75f"
+  "gitHead": "fc90fc875462421c1faa35862ac4bc436de8e75f",
+  "dependencies": {
```

Comment:
These should be `devDependencies`

Reply:
Fixed ✅ 
Just wondering, why aren't u using `devDependencies` convention on other packages in the repo as well (it used only here)?

Reply:
We're in the middle of a bigger migration to `tsup` with all the packages, and one of the steps in that migration is moving most if not all dependencies to devDependencies. We're not done yet, maybe that's what you're seeing.

https://github.com/storybookjs/storybook/issues/18732